import sys, os

print("fail")
sys.exit(1)